unitTests/ui/ResultsPerPageTest.ts

```diff
@@ -114,6 +114,21 @@ export function ResultsPerPageTest() {
         expect(test.env.queryController.options.resultsPerPage).toBe(15);
       });
 
+      it('choicesDisplayed links display the right aria-label', () => {
+        test = Mock.optionsComponentSetup<ResultsPerPage, IResultsPerPageOptions>(ResultsPerPage, {
+          choicesDisplayed: [10, 25, 50]
+        });
+        Simulate.query(test.env, {
+          results: FakeResults.createFakeResults(1000)
+        });
+
+        const anchors = $$(test.cmp.element).findAll('a.coveo-results-per-page-list-item-text');
+        expect($$(anchors[0]).text()).toBe('10');
+        expect(anchors[0].parentElement.getAttribute('aria-label')).toBe('Display 10 results per page');
```

Comment:
While it does not hurt to check more than one case, I think just checking one, whether the `10` or `50`, is enough. Same for the Pager test, either Page 1 or Page 3. It makes the test more concise, with less redundancy.

Reply:
Done ✅